src/BibleWell.App/App.axaml.cs

```diff
@@ -193,6 +199,58 @@ private ServiceProvider ConfigureServiceProvider(IConfiguration configuration)
         return services
             .BuildServiceProvider();
     }
+    /// <summary>
+    /// Shuts down the application.
+    /// </summary>
+    public void Shutdown()
```

Comment:
This isn't used in the PR but it took me a while to figure out how to do this when I was working on another possible solution so I left the method in here.